packages/London-Boroughs 3 (Corrected to 32 boroughs)/README.md

```diff
@@ -0,0 +1,50 @@
+# London, Boroughs
+
+[![License: Unlicense](https://img.shields.io/badge/license-Unlicense-blue.svg)](http://unlicense.org/)
+
+Contains all the 33 boroughs of London:
```

Comment:
Are there 33 or 32 boroughs? 🤔 
I would prefer to publish only 1 package with the right data.

Moreover all the previous comments apply to this package too.